src/main/resources/application.yml

```
@@ -10,7 +10,4 @@ spring:
 
 jwt:
   secret:
-    key: ${JWT_SECRET_KEY}
-  refresh:
-    secret:
-      key: ${JWT_REFRESH_SECRET_KEY}
\ No newline at end of file
+    key: ${JWT_SECRET_KEY}
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)